views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Map.js

```diff
@@ -49,6 +49,7 @@ define([
         return parseInt(value) + 1;
     });
     let uidCounter = 0;
+    const defaultScore = 1;
```

Comment:
why 1? maybe response.getMappingAttribute('defaultValue')?

Reply:
On the ticket, they say the default here must be 1. The response default is 0